src/druglab/pipe/blocks/prepare.py

```
@@ -0,0 +1,246 @@
+                        ff_obj.Minimize(maxIts=self.max_iters)
+                        energies[cid] = ff_obj.CalcEnergy()
+
+            elif self.ff == "UFF":
+                for cid in conf_ids:
+                    ff_obj = AllChem.UFFGetMoleculeForceField(mol, confId=cid)
+                    if ff_obj is None:
+                        continue
+                    ff_obj.Minimize(maxIts=self.max_iters)
+                    energies[cid] = ff_obj.CalcEnergy()
+
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)